contracts/BtcLightClient.template

```diff
@@ -281,7 +281,7 @@ contract BtcLightClient is ILightClient, System, IParamSubscriber{
           3. else if count in half-open interval (2*maxWeight, 2.75*maxWeight] => return 3*maxWeight - count
           4. else return count/4
    */  
-  function calculateRelayerWeight(uint256 count) public view returns(uint256) {
+  function calculateRelayerWeight(uint256 count) public view returns(uint256) { //@openissue
```

Comment:
I assume that the plotting of the count to weight values is by design (which I do not have).
still the jump of weight value between count = (2*maxWeight + 3*maxWeight/4) and the next value: count = 1 + (2*maxWeight + 3*maxWeight/4) is a bit worrying. having 'jumps' in revenue calculation is usually a place where a hacker can play the system. 
any idea why is that?